src/hivpy/prep.py

```diff
@@ -130,6 +130,8 @@ def get_suspect_risk_pop(self, pop: Population):
                                    COND(col.LTP_HIV_STATUS, op.eq, True),
                                    COND(col.R_PREP, op.lt, self.prob_suspect_risk_prep)))
 
+    # FIXME: this function may be removed if there are no issues with
+    # updating PrEP after HIV diagnosis during population evolution
```

Comment:
Finding this out would include testing multiple modules together; where would it make the most sense to put this test?

Reply:
That's a good question; and it's tricky to say right now because we're not sure _what_ we would need to test yet to be able to figure this out. It might be possible to define some integration tests in a file of their own if we can say definitively what some behaviour of the simulation as a whole should look like when working together.